saleor/order/migrations/0172_auto_20230627_0816.py

```diff
@@ -1,23 +1,48 @@
 # Generated by Django 3.2.19 on 2023-06-01 08:15
 
-from django.apps import apps as registry
 from django.db import migrations, models
-from django.db.models.signals import post_migrate
+from django.db.models import Exists, OuterRef
 
-from .tasks.saleor3_15 import drop_status_field_from_transaction_event_task
+# Batch size of size 5000 is about 5MB memory
+BATCH_SIZE = 5000
+
+
+def queryset_in_batches(qs, batch_size: int):
+    def batch_ids():
+        length = len(ids)
+        for index in range(0, length, batch_size):
+            yield ids[index : min(index + batch_size, length)]
+
+    ids = qs.values_list("id", flat=True)
```

Comment:
We should not load all objects at once into memory, it may cause troubles when migrating.